docs/components.md

```diff
@@ -81,16 +89,19 @@ Ionic apps are made of high-level building blocks called Components, which allow
   <p>Inputs provides a way for users to enter data in your app.</p>
 </DocsCard>
 
-<DocsCard header="Item" href="api/item" icon="/icons/component-item-icon.png">
-  <p>Items are an all-purpose UI container that can be used as part of a list.</p>
+<DocsCard header="Item" href="api/item" img="/icons/feature-component-item-icon.png">
+  <p>A common UI paradigm that serves as an entry point to more detailed information.</p>
```

Comment:
Same with this one. I kinda preferred the original description. Is there a reason it changed?